concepticondata/conceptlists/Carling-2017-240c.tsv

```diff
@@ -1,241 +1,240 @@
+Carling-2017-240c-15	15	1630	MACAW	macaw	4007	Wild animals	
+Carling-2017-240c-16	16	882	PARROT	parrot	4008	Wild animals	
+Carling-2017-240c-17	17	308	TOUCAN	toucan	4009	Wild animals	
+Carling-2017-240c-18	18	315	CURASSOW	mutum, curassow	4010	Wild animals	
+Carling-2017-240c-19	19	2438	CAIMAN	caiman	4011	Wild animals	
+Carling-2017-240c-20	20	227	FISH	fish (n)	12151	Wild animals	
+Carling-2017-240c-21	21	843	SPIDER	spider	12340	Wild animals	
+Carling-2017-240c-22	22	1791	BUTTERFLY	butterfly	12341	Wild animals	
+Carling-2017-240c-23	23	314	CAPYBARA	capybara	12342	Wild animals	
+Carling-2017-240c-24	24	730	SNAKE	snake	12343	Wild animals	
+Carling-2017-240c-25	25	587	ANT	ant	12344	Wild animals	
+Carling-2017-240c-26	26			margay	12345	Wild animals	
+Carling-2017-240c-27	27	2640	LARVA	larva	12346	Wild animals	
+Carling-2017-240c-28	28			Brazilian tortoise	12347	Wild animals	
+Carling-2017-240c-29	29	1350	MONKEY	monkey	12348	Wild animals	
+Carling-2017-240c-30	30	1793	BAT	bat	12349	Wild animals	
+Carling-2017-240c-31	31	316	PACA	paca	12350	Wild animals	
+Carling-2017-240c-32	32	436	WHITE-LIPPED PECCARY	white-lipped peccary	12351	Wild animals	
+Carling-2017-240c-33	33	435	COLLARED PECCARY	collared peccary	12352	Wild animals	
+Carling-2017-240c-34	34	1490	RAT	rat	12353	Wild animals	
+Carling-2017-240c-35	35	1936	DEER	deer	12354	Wild animals	
+Carling-2017-240c-36	36	313	AGOUTI	agouti	12355	Wild animals	
+Carling-2017-240c-37	37			guan bird	12356	Wild animals	
+Carling-2017-240c-38	38	309	OPOSSUM	opossum	12357	Wild animals	
+Carling-2017-240c-39	39			tinamou bird	12358	Wild animals	
+Carling-2017-240c-40	40			white-throated tinamou	12359	Wild animals	
+Carling-2017-240c-41	41	937	BIRD	bird	12376	Wild animals	
+Carling-2017-240c-42	42	1430	STAR	star	4012	Celestial bodies	
+Carling-2017-240c-43	43	1343	SUN	sun	4013	Celestial bodies	
+Carling-2017-240c-44	44	1313	MOON	moon	4014	Celestial bodies	
+Carling-2017-240c-45	45	948	WATER	water (n)	4015	Physical/natural world	
+Carling-2017-240c-46	46	1228	EARTH (SOIL)	earth (soil, ground)	12137	Physical/natural world	
+Carling-2017-240c-47	47	141	BURN (SOMETHING)	to burn (tr.)	12138	Physical/natural world	
+Carling-2017-240c-48	48	906	TREE	tree	12207	Physical/natural world	
+Carling-2017-240c-49	49	646	ASH	ash(es)	12208	Physical/natural world	
+Carling-2017-240c-50	50	778	SMOKE (EXHAUST)	smoke (n)	12209	Physical/natural world	
+Carling-2017-240c-51	51	221	FIRE	fire	12210	Physical/natural world	
+Carling-2017-240c-52	52	639	MOUNTAIN	mountain	12211	Physical/natural world	
+Carling-2017-240c-53	53		*SHORE OR BANK	shore, bank	12212	Physical/natural world	
+Carling-2017-240c-54	54	1474	SEA	sea	12213	Physical/natural world	
+Carling-2017-240c-55	55	645	OCEAN	ocean	12214	Physical/natural world	
+Carling-2017-240c-56	56	624	LAKE	lake	12215	Physical/natural world	
+Carling-2017-240c-57	57		*FLOWING BODY OF WATER	river, stream, brook	12216	Physical/natural world	
```

Comment:
I remember we had this problem in the past, that we have similar concept lists with this, like "river, stream". When adding the concept, make sure to remap them as well.